### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by githamdaan25**
added hamdaan

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -300,6 +300,25 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
             </div>
           </div>
         </div>
+      </div>
+       <div class="col-md-4">
+        <div class="profile-section">
+          <div class="text-center">
+            <img src="./assets/images/hamdaan.jpg" alt="Sandeep Velayudhan" class="profile-image">
```
Comment (ShehrozIrfan) [inline index.html:307]: @githamdaan25 
Is the `alt` attribute value is correct?

**Comment by ShehrozIrfan** [comment]
@githamdaan25 
Good work!
Please check again the `alt` attribute value, it seems that needs to be corrected.

**Comment by githamdaan25** [comment]
Correcting it

File: index.html (hunk 1 above)
Comment (githamdaan25) [reply index.html]: corrected it

**Comment by ShehrozIrfan** [comment]
@githamdaan25 
Great.
Your PR is merged.
You can follow me and star this repository for further updates.
Have fun!

**Comment by ShehrozIrfan** [comment]
Hey there!
If your contribution is excluded for this repository, then please visit the repository [Ruby Rocks](https://github.com/ShehrozIrfan/ruby-rocks/) for contribution to hacktoberfest. It contains a plenty of beginner's friendly issues accepted for hacktoberfest.

Have fun!

**Pull request merged**